src/mots/cli.py

```diff
@@ -253,6 +257,29 @@ def search(args: argparse.Namespace):
         print(out)
 
 
+def check_for_updates(args: argparse.Namespace) -> None:
+    """
+    Show a message if there is a newer version available.
+
+    This method checks the RSS feed on PyPI.
```

Comment:
You should use the `simple` API for this - I've added similar code to `moz-phab` already.

Reply:
I've considered this (I was reading that documentation around the same time you had that PR up for mozphab) -- I don't see anything hinting that this feed will be deprecated (unlike the old JSON endpoint that you were originally using in that PR).

I do agree that it would be nicer to use JSON vs. XML (and an actual API endpoint) though I think this does simplify the parsing of version names significantly. I'm leaning towards making this change in the future when I have a more evolved version checking (and possibly updating) system.